lib/resources/xml.rb

```diff
@@ -20,7 +20,16 @@ def parse(content)
     end
 
     def value(key)
-      REXML::XPath.each(@params, key.first.to_s).map(&:text)
+      output = []
+      REXML::XPath.each(@params, key.first.to_s) do |element|
+        if element.is_a?(REXML::Attribute)
+          output.push(element.to_s)
+        elsif element.is_a?(REXML::Element)
```

Comment:
Should this be a default `else`? Is there any chance we'd stumble across something that's not a `REXML::Attribute` or `REXML::Element` that wouldn't respond to a `#text` method call?

Reply:
@adamleff I cannot say with 100% confidence.  I believe the only remaining type would be context which contains data about parsing.

I think a default statement would be nice, but it should fall into `ResourceSkipped` or `ResourceFailed` so that it doesn't run into the same issue that I am fixing with this PR.  That would allow it to sanely fail rather than with a MethodError.

Reply:
I would be in support of that!